frontend/src/tasks/tasks.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+const template = require('./tasks.html');
+const api = require('../api');
+
+module.exports = app => app.component('tasks', {
+  data: () => ({
+    status: 'init',
+    tasks: [],
+    selectedRange: 'all',
```

Comment:
Should default to "today". "all time" is way too much data and way too slow to be the default.